### PR TITLE
Correct spelling and wording

### PR DESCRIPTION
**Pull request by davidgraeff**

### CODE REVIEW

File: convention.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ Controllers are free to restrict discovery to a specific root topic, configurabl
 
 As soon as a device starts to publish any Homie related topic,
 it MUST finish with all topics within a timeframe of 500ms.
-Controller can assume that not received topics within this timeframe are not set.
+Controllers should assume an empty string for topics values not received within this timeframe.
```
Comment (ThomDietrich) [inline convention.md]:
```suggestion
Controllers should assume the default for topic values not received within this timeframe.
```

File: convention.md [hunk 2]
```diff
@@ -320,6 +320,11 @@ Any other topic is not part of the Homie convention.
 
 ## Extensions
 
-In addition to homie-core the device may support multiple extensions which are defined in separate conventions.
-Every extension is identified by a unique id.
-The id consists of the reverse domain name and a freely choosen prefix. For example a company `example.org` that likes to extend homie a cool feature would choose `org.example.cool-feature`. The prefix `homie` is reserved.
+This convention only covers discoverability of devices and its capabilities.
+The aim is to have standardized MQTT topics for all kind of complex scenarios.
+A Homie device may therefore support extensions, defined in separate documents.
+Every extension is identified by a unique id and will be linked from the section.
```
Comment (ThomDietrich) [inline convention.md]:
```suggestion
Every extension is identified by a unique ID and will be linked from the section.
```

File: convention.md [hunk 3]
```diff
@@ -320,6 +320,11 @@ Any other topic is not part of the Homie convention.
...
+
+The id consists of the reverse domain name and a freely choosen prefix. The domain and prefix `homie` is reserved.
```
Comment (ThomDietrich) [inline convention.md]:
```suggestion
The ID consists of the reverse domain name and a freely chosen prefix.
The domain and prefix `homie` is reserved.
```

File: convention.md [hunk 4]
```diff
@@ -320,6 +320,11 @@ Any other topic is not part of the Homie convention.
...
+
+For example an organisation `example.org` wants to add a feature `our-feature`. They would choose the extension id `org.example.our-feature`.
```
Comment (ThomDietrich) [inline convention.md]:
```suggestion
For example, an organization `example.org` wanting to add a feature `our-feature` would choose the extension ID `org.example.our-feature`. 
```

**Review by ThomDietrich — APPROVED** [review]

File: convention.md (hunk 1 above)
Comment (ThomDietrich) [reply convention.md]: Up for discussion. Should we distinguish between mandatory and optional attributes?

File: convention.md [hunk 5]
```diff
@@ -320,6 +320,12 @@ Any other topic is not part of the Homie convention.
...
 
-In addition to homie-core the device may support multiple extensions which are defined in separate conventions.
-Every extension is identified by a unique id.
-The id consists of the reverse domain name and a freely choosen prefix. For example a company `example.org` that likes to extend homie a cool feature would choose `org.example.cool-feature`. The prefix `homie` is reserved.
+This convention only covers discoverability of devices and its capabilities.
+The aim is to have standardized MQTT topics for all kind of complex scenarios.
+A Homie device may therefore support extensions, defined in separate documents.
+Every extension is identified by a unique ID and will be linked from the section.
+
+The ID consists of the reverse domain name and a freely chosen prefix.
```
Comment (Thalhammer) [inline convention.md]: I know I wrote the original, but noticed it just now.
Isn't this a postfix not a prefix ?

**Review by Thalhammer — CHANGES_REQUESTED** [review]

File: convention.md (hunk 5 above)
Comment (davidgraeff) [reply convention.md]: suffix is the word you are looking for ^^

File: convention.md [hunk 6]
```diff
@@ -323,9 +323,9 @@ Any other topic is not part of the Homie convention.
 This convention only covers discoverability of devices and its capabilities.
 The aim is to have standardized MQTT topics for all kind of complex scenarios.
 A Homie device may therefore support extensions, defined in separate documents.
-Every extension is identified by a unique ID and will be linked from the section.
+Every extension is identified by a unique ID and will be linked from this section.
 
-The ID consists of the reverse domain name and a freely chosen prefix.
-The domain and prefix `homie` is reserved.
+The ID consists of the reverse domain name and a freely chosen suffix.
+The domain and suffix `homie` is reserved.
```
Comment (Thalhammer) [inline convention.md:329]: Well `homie` was really a prefix, isn't it ?
Only the first time it should be suffix.

Comment (ThomDietrich) [reply convention.md:329]: See #147

**Pull request merged**